code/modules/antagonists/nukeop/equipment/pinpointer.dm

```diff
@@ -19,17 +19,18 @@
 		if(bomb.timing)
 			. += "Extreme danger. Arming signal detected. Time remaining: [bomb.get_time_left()]."
 
-/obj/item/pinpointer/nuke/process()
+/obj/item/pinpointer/nuke/process(delta_time)
 	..()
-	if(active) // If shit's going down
-		for(var/obj/machinery/nuclearbomb/bomb in GLOB.nuke_list)
-			if(bomb.timing)
-				if(!alert)
-					alert = TRUE
-					playsound(src, 'sound/items/nuke_toy_lowpower.ogg', 50, FALSE)
-					if(isliving(loc))
-						var/mob/living/L = loc
-						to_chat(L, "<span class='userdanger'>Your [name] vibrates and lets out a tinny alarm. Uh oh.</span>")
+	if(!active || alert)
+		return
+	for(var/obj/machinery/nuclearbomb/bomb as anything in GLOB.nuke_list)
+		if(!bomb.timing)
+			return
```

Comment:
wouldn't a continue be better here? i don't know if GLOB.nuke_list can have multiple bombs, but if it can why not check them too?

Reply:
true